src/components/reusables/SummaryPage/SummaryPage.tsx

```diff
@@ -34,13 +38,15 @@ export interface SummaryPageProps {
   successfulDelegations?: ApiDelegation[];
   restartProcessPath: string;
   pageHeaderText: string;
+  headerIcon: React.ReactNode;
+  headerColor?: PageColor;
   topListText?: string;
   bottomListText?: string;
   bottomText?: string;
-  mainButton?: React.ReactNode;
-  complementaryButton?: React.ReactNode;
-  headerIcon: React.ReactNode;
-  headerColor?: PageColor;
+  nextButtonClick?: () => void;
+  nextButtonDisabled?: boolean;
+  nextButtonLoading?: boolean;
```

Comment:
Why are these props called nextButton when they always refer to a confirmation button?
Can't they be called confirmationButton, so it's more clear that that's what they are? 😅